includes/handler/class-move.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * Move handler file.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Handler;
+
+use Activitypub\Http;
+use Activitypub\Collection\Followers;
+
+use function Activitypub\is_actor;
+use function Activitypub\object_to_uri;
+
+/**
+ * Handle Move requests.
+ *
+ * @see https://www.w3.org/TR/activitystreams-vocabulary/#dfn-move
+ * @see https://docs.joinmastodon.org/user/moving/
+ * @see https://docs.joinmastodon.org/spec/activitypub/#Move
+ */
+class Move {
+	/**
+	 * Initialize the class, registering WordPress hooks.
+	 */
+	public static function init() {
+		\add_action( 'activitypub_inbox_move', array( self::class, 'handle_move' ) );
+	}
+
+	/**
+	 * Handle Move requests.
+	 *
+	 * @param array $activity The JSON "Move" Activity.
+	 */
+	public static function handle_move( $activity ) {
+		$target = self::extract_target( $activity );
+		$origin = self::extract_origin( $activity );
+
+		if ( ! $target || ! $origin ) {
+			return;
+		}
+
+		$target_object = Http::get_remote_object( $target );
+		$origin_object = Http::get_remote_object( $origin );
+
+		$verified = self::verify_move( $target_object, $origin_object );
+
+		if ( ! $verified ) {
+			return;
+		}
+
+		$target_follower = Followers::get_follower_by_actor( $target, false );
+		$origin_follower = Followers::get_follower_by_actor( $origin, false );
```

Comment:
```suggestion
		$target_follower = Followers::get_follower_by_actor( $target );
		$origin_follower = Followers::get_follower_by_actor( $origin );
```
`get_follower_by_actor()` doesn't have a second arg.